diff --git i/configure.ac w/configure.ac
index 4d32c907..0db3c39a 100644
--- i/configure.ac
+++ w/configure.ac
@@ -192,6 +192,14 @@ case "$host" in
 		with_sgen_default_concurrent=yes
 		LN_S=cp
 		;;
+	arm*-vita*-eabi*)
+		dnl PS Vita: POSIX 信号后端 (sigaction 由 vita-shim 提供)
+		use_sigposix=yes
+		with_sigaltstack=no
+		with_tls=pthread
+		with_sgen_default_concurrent=yes
+		need_link_unlink=yes
+		;;
 	*-*-*netbsd*)
 		CPPFLAGS="$CPPFLAGS -D_REENTRANT -DGC_NETBSD_THREADS -D_GNU_SOURCE"
 		libmono_cflags="-D_REENTRANT"
@@ -4551,6 +4559,15 @@ case "$host" in
 			;;
 		esac
 		;;
+	arm*-vita*-eabi*)
+		TARGET=ARM;
+		arch_target=arm;
+		ACCESS_UNALIGNED="no"
+		AOT_SUPPORTED="yes"
+		CPPFLAGS="$CPPFLAGS -D__ARM_EABI__"
+		with_sigaltstack=no
+		use_sigposix=yes
+		;;
 	arm*-netbsd*-eabi*)
 		TARGET=ARM;
 		arch_target=arm;
@@ -4654,6 +4671,10 @@ if test "x$host" != "x$target"; then
 		TARGET=ARM;
 		CPPFLAGS="$CPPFLAGS -D__ARM_EABI__"
 		;;
+   arm*-vita*-eabi*)
+		TARGET=ARM;
+		CPPFLAGS="$CPPFLAGS -D__ARM_EABI__"
+		;;
    arm*-netbsd*-eabi*)
 		TARGET=ARM;
 		CPPFLAGS="$CPPFLAGS -D__ARM_EABI__"
